src/app/shared/model/Config.ts

```diff
@@ -30,7 +30,7 @@ export interface Viewable {
 export class GroupConfig {
   projects?: Projects;
   members?: Members;
-  sshKeys?: SSHKeys;
+  sshkeys?: SSHKeys;
```

Comment:
This is a "not so nice" part. But as we already wanted to change the whole role management in the ui, this is probably the fastest "solution" as we use a shared "View" enum now for auth & the page title. All urls are lowerCase...

Reply:
I think that this actually fixes it. Auth guard does not use case insensitive comparison, so for ssh keys it should not work currently.